Steps/BuildPlatform.yml

```diff
@@ -128,14 +128,14 @@ steps:
 - task: CmdLine@2
   displayName: Build
   inputs:
-    script: stuart_build -c ${{ parameters.build_file }} TOOL_CHAIN_TAG=${{ parameters.tool_chain_tag}} TARGET=${{ parameters.build_target}} -a ${{ parameters.build_arch}} ${{ parameters.build_flags}}
+    script: stuart_build -c ${{ parameters.build_file }} TOOL_CHAIN_TAG=${{ parameters.tool_chain_tag}} TARGET=${{ parameters.build_target}} ${{ parameters.build_flags}}

```

Comment:
I understand the platform can now pass `-a <ARCH>`, but doesn't this leave `parameters.build_arch` unused in this command? That seems unexpected given its a parameter to the template.

Reply:
@makubacki the setup and update commands still have the -a flag passed to them. I'm not sure how necessary that is. If you don't think that is necessary for the BuildPlatform.yml, I'm happy to remove them there too

Reply:
I'd prefer dropping the parameter entirely and marking this as a breaking change. I think it cleans up how parameters behave.

Reply:
Dropped the parameter and specified this as a breaking change.

Reply:
I will roll this as a 10.0.0 release.